community/function/src/main/java/org/neo4j/function/Predicate.java

```diff
@@ -24,7 +24,8 @@
  *
  * @param <T> the type of the input to the predicate
  */
-public interface Predicate<T> extends ThrowingPredicate<T,RuntimeException>
+@Deprecated // Use java.util.function.Predicate instead
```

Comment:
please add `@deprecated See {@link java.util.function.Predicate} instead.` or similar into a javadoc instead of a comment.